OPCB/functions/retrieve_help1.sqf

```diff
@@ -1,53 +1,57 @@
-	_civ = _this select 0;
-	_theFugitive = _this select 3 select 0;
+params ["_civ", "_caller", "_actionId", "_arguments"];
+_arguments params ["_theFugitive"];
 
-	_questionedFugitive = _civ getVariable "isQuestionedFugitive";
-	_isFugitive = _civ getVariable "isFugitive";
+_questionedFugitive = _civ getVariable "isQuestionedFugitive";
+_isFugitive = _civ getVariable "isFugitive";
 
-	if(_isFugitive == "yes") then
-	{
-		hint "YOU WON'T GET ME ALIVE";
-		_enemySide = createGroup resistance;
-		removeAllWeapons _civ;
-		[_civ] joinSilent grpNull;
-		[_civ] joinSilent _enemySide;
+if(_isFugitive == "yes") then {
+	removeAllActions _civ;
+	hint "YOU WON'T GET ME ALIVE";
+	_enemySide = createGroup resistance;
+	removeAllWeapons _civ;
+	[_civ] joinSilent grpNull;
+	[_civ] joinSilent _enemySide;
 
-		_civ addVest "V_TacVest_blk_POLICE";
-		_civ addWeapon "hgun_ACPC2_F";
-		_civ addMagazineCargo ["9Rnd_45ACP_Mag",1];
-		_civ enableAI "MOVE";
-		_civ action ['SwitchWeapon', _civ, _civ, 0];
-		_enemySide setCombatMode "COMBAT";
-	}
-	else
-	{
-		if (_questionedFugitive == "no") then
-		{
-			_civ setVariable ["isQuestionedFugitive", "yes", true];
-		 	_fugitiveAnswers = ["IDK","IDC","NO","YES","IDU"];
-			_fugiAnswer = _fugitiveAnswers call BIS_fnc_selectRandom;
+	_civ addVest "V_TacVest_blk_POLICE";
+	_civ addWeapon "hgun_ACPC2_F";
+	_civ addMagazineCargo ["9Rnd_45ACP_Mag",1];
+	_civ enableAI "MOVE";
+	_civ action ['SwitchWeapon', _civ, _civ, 0];
+	_civ setUnitPos "UP";
+	_enemySide setCombatMode "COMBAT";
+	_civ reveal _caller;
+	_civ doFire _caller;
```

Comment:
I don't want to write code for an enemy to shoot at you. I want this guy to be an AI enemy. Like turn it into an actual enemy

Reply:
In that case he's going to pull out a gun and either start running in some direction, or just pause for a second with the gun in his hand while he gets gunned down immediately by players. I'd be surprised if he manages to do anything else.

Reply:
You could do something like this for interesting results: make him run away without pulling out a gun, and if he manages to get far enough away, then he could pull out a gun. He might even try to get behind some cover first or something.

Would be even more interesting if there were multiple possibilities with different random outcomes for replayability.

Reply:
Make an issue about this. I don't have the best experience with this kind of unit manipulation. It would take up too much of my time to play with the mechanics. We should let this go in this PR